### ISSUE TITLE
Error message mismatches for entering invalid Email id for user under "Sends the user a new verification email"

### ISSUE THREAD

**Issue by faznan3nazer** [anitab-org/mentorship-backend#720]
**Describe the bug**

When we enter an invalid mail id, an appropriate error message should be shown and listed under the Responses.

**To Reproduce**

1. Go to 'user-Sends the user a new verification email'
2. Provide the invalid email id 
3. {
      "email": "dssfgdsgkhu"
   }
4. Click on 'Execute'.
5.  Scroll down to 'Responses'.
6. 
![Screenshot (277)](https://user-images.githubusercontent.com/50888936/89913705-a809f100-dc11-11ea-95a1-ef0f92b39d2a.png)

**Expected behavior**
That error message should be listed in the responses.

**Desktop (please complete the following information):**
 - OS: windows10
 - Browser chrome

**Additional context**
The error message generated should match with the error message listed under responses.

**Comment by saikirankv** [comment]
@isabelcosta  can I take this issue?

**Comment by isabelcosta** [comment]
Yes @saikirankv :) Go ahead, and let us know if you have any doubt

**Comment by saikirankv** [comment]
PR: https://github.com/anitab-org/mentorship-backend/pull/751
@isabelcosta  I have submitted a PR, Tests are failing with below error
Merging #751 into develop will not change coverage.
The diff coverage is 50.00%.

can you help me with this?

**Comment by saikirankv** [comment]
@isabelcosta  can you help me with this?

**Comment by saikirankv** [comment]
@isabelcosta here is my PR https://github.com/anitab-org/mentorship-backend/pull/765
tests are passing, what is the process for PR merge.

**Comment by SanketDG** [comment]
@saikirankv Someone will review your PR soon

**Comment by harshakhmk** [comment]
I would like to work on this issue, it would be a perfect start for a beginner like me @SanketDG @isabelcosta

**Comment by epicadk** [comment]
@isabelcosta I think that 400 invalid input pretty much covers all such cases

**Comment by epicadk** [comment]
@harshakhmk you are already assigned a different issue. Please work on one issue at a time.

**Comment by epicadk** [comment]
> @isabelcosta I think that 400 invalid input pretty much covers all such cases

Will add status available after discussing this in the open session.

**Comment by harshakhmk** [comment]
@epicadk   previous issue was not clear to proceed, I got stuck in the middle, meanwhile I can work on this

**Comment by epicadk** [comment]
In anycase I think we need to discuss this issue in the open session before moving forward.
